test/Parse/trailing-comma.swift

```diff
@@ -0,0 +1,61 @@
+// RUN: %target-typecheck-verify-swift -disable-experimental-parser-round-trip -enable-experimental-feature TrailingComma
+
+func testTupleAndArgumentsWithTrailingComma() {
+    let _ = (a: 1, b: 2, c: 3,)
+
+    func foo(a: Int = 0, b: Int = 0, c: Int = 0,) -> Int {
+        return a + b + c
+    }
+
+    let _ = foo(a: 1, b: 2, c: 3,)
+}
+
+func testConditionsWithTrailingComma() {
+    func f(_ block: (Bool) -> Bool) -> Bool { block(true) }
+
+    if true, { }
```

Comment:
What about cases such as `if true { let a = 1 }` how would those be handled? Looking through `isStartOfConditionalStmtBody` method, I couldn't see if those would be handled or not.

Reply:
It parses fine because `skipSingle()` jumps from `{` to `}` and then there's nothing after that rules out `{ let a = 1 }` as the if body.